src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double AvgCurrent;
+    double prevCurrent;
+    double changeTime;
+    QTime prevTime;
+    QTime currentTime;
+    double AmpHours;
+    double existingCurrent;
+    double dTime;
```

Comment:
In general, avoid abbreviations, this d is a good example of where not to abbreviate, write the full `delta` for this variable.